### PR TITLE
:memo: Add graphical library benchmark (EPI-16)

### PR DESCRIPTION
**Pull request by AubaneNourry**
I added a .md file with the performance comparison of three graphical libraries: SFML, SDL2 and Allegro, in order to benchmark which one we will use for our project.

In summary, SDL2 is the best choice for the project.

### CODE REVIEW

**Review by 6im0n — APPROVED** [review]
Good for me !

File: documentation/benchmarks/graphical_library_benchmark.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  - Allegro demonstrates excellent frame times and really low CPU usage, but with comparable memory usage to SFML.
+- **Scrolling Background:** 
+  - SFML and SDL2 have similar frame times and are capable of handling scrolling backgrounds smoothly.
+  - Allegro exhibits a significant range in frame time, which can be attributed to its rendering method. While it has the lowest CPU usage, its performance is less consistent.
+### Pros and Cons
+
+| **Library**   | **Pros**                                   | **Cons**                                   |
+|---------------|--------------------------------------------|--------------------------------------------|
+| **SFML**      | Easy to use, good documentation            | Less optimized for low-end hardware        |
+| **SDL2**      | Highly portable, strong community support  | API complexity for beginners               |
```
Comment (axel-eck) [inline documentation/benchmarks/graphical_library_benchmark.md:75]: Why Allegro is missing from this table ?

**Review by axel-eck — CHANGES_REQUESTED** [review]
I think you forgot on row in your cons / pros table.
Moreover, I think you should give the tests codes to ensure no bias towards any of the technologies. If you used a benchmarking solution, and not your own code, can you mention which one.

**Comment by axel-eck** [comment]
Be careful for next time but your branch should have been named 'feature/epi-16-graphic-libraries-benchmark' in order to comply GitFlow rules. You can use the button or the shortcut directly in linear to get the proper branch name.
Screenshot 2024-09-18 at 16 48 49
_Now that the issue is linked, the button gives you the current branch name but for next time just use the generated name._

**Comment by AubaneNourry** [comment]
Update: Branch renamed, check the new PR #2

**Pull request closed without merging**